conbench/entities/hardware.py

```diff
@@ -188,9 +188,27 @@ class MachineCreate(marshmallow.Schema):
 
 
 class ClusterCreate(marshmallow.Schema):
-    name = marshmallow.fields.String(required=True)
-    info = marshmallow.fields.Dict(required=True)
-    optional_info = marshmallow.fields.Dict(required=True)
+    name = marshmallow.fields.String(
+        required=True,
+        metadata={
+            "description": "Information related to cluster (e.g. `hosts`, `nodes` or `number of workers`)"
```

Comment:
I think there is a trailing space required before the closing `"` in this line.

Not just here, in all of those lines where the sentence continues in the next line.